docs/source/tutorials/mint-your-first-asset.md

```diff
@@ -0,0 +1,90 @@
+# Minting your first Asset
+
+When you use Iroha in your application you will definitely face the need to mint an asset.
+Use this tutorial to master Iroha Special Instructions in general and `MintAsset` in particular.
+
+## Prerequisites
+
+* [Rust](https://www.rust-lang.org/tools/install)
+* Text Editor or IDE
+* Existing Rust project //TODO: create some sandbox project
+
+## Steps
+
+### 1. Add dependency on Iroha crate
+
+Inside `path_to_your_project/Cargo.toml` add a new dependency:
+
+```toml
+iroha = "2.0.0"
+```
+
+### 2. Find a place where you need to mint an asset
+
+Usually any business operation will have a need to mint an asset:
+
+* Uploading of digital documents
+* Withdraw of crypto currencies
+* Etc.
+
+Look at your project and find a good place to put it, which will encapsulate Iroha related logic inside
+and may be easily modified if needed in future.
+
+### 3. Use Iroha CLI Client to prepare Iroha Peer
+
+Iroha Special Instructions executed on behalf of an authority - Account.
+
+If you already has an account to store assets on - feel free to skip these step.
+If not - you will need to receive Account's Key Pair with permissions to Register an Account.
+
+TL;DR - after [configuration of Iroha CLI](https://github.com/hyperledger/iroha/blob/iroha2-dev/iroha_client_cli/README.md)
+run this command:
+
+```bash
+./iroha_client_cli account register --domain="my_domain" --name="my_account" --key="{account_public_key}"
+```
+
+### 4. Construct Iroha Special Instruction
```

Comment:
I think it should be mentioned that we also can submit mint ISI via CLI Client, but the purpose here is to show how to do it with code